manifests/infra/mysql-o11y/base/cronjob.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: mysql-o11y
+  namespace: monitoring
+spec:
+  schedule: "*/1 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: mysql
+            image: mysql:8.0
+            args: ["mysqldump --single-transaction --quick --set-charset -u $RDS_USER -p $RDS_PASSWORD -h $RDS_HOST dreamkast conference_days conferences proposal_item_configs proposal_items proposals talk_categories talk_difficulties talk_times talks talks_speakers tracks stats_of_registrants viewer_counts chat_messages > $HOME/rds.dump; mysql -u $MYSQL_USER -p $MYSQL_PASSWORD  -h $MYSQL_HOST dreamkast < $HOME/rds.dump"]
```

https://github.com/cloudnativedaysjp/dreamkast-infra/pull/1299#discussion_r790832166 `chat_messages`は別途検討しましょうか。
```suggestion
            args: ["mysqldump --single-transaction --quick --set-charset -u $RDS_USER -p $RDS_PASSWORD -h $RDS_HOST dreamkast conference_days conferences proposal_item_configs proposal_items proposals talk_categories talk_difficulties talk_times talks talks_speakers tracks stats_of_registrants viewer_counts > $HOME/rds.dump; mysql -u $MYSQL_USER -p $MYSQL_PASSWORD  -h $MYSQL_HOST dreamkast < $HOME/rds.dump"]
```

---

別コメントでもかきましたが、chat_messages は諦めます。お騒がせしました。